src/main/java/ro/hiringsystem/model/CandidateUser.java

```diff
@@ -0,0 +1,25 @@
+package ro.hiringsystem.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import ro.hiringsystem.model.abstracts.User;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Optional;
+
+@SuperBuilder
+@Getter
+@Setter
+public class CandidateUser extends User {
+
+    private File cv;
```

Comment:
I think this field should have its separate object type that would contain a file but also separate fields for work experience, academic background, etc, so we may be able to search CV info by keyword in the future.